cmd/cnab-run/env.go

```diff
@@ -35,9 +37,24 @@ func setupDockerContext() (command.Cli, error) {
 	if err != nil {
 		return nil, err
 	}
-	return cli, cli.Initialize(&cliflags.ClientOptions{
+	if err := cli.Initialize(&cliflags.ClientOptions{
 		Common: &cliflags.CommonOptions{
 			Context: "cnab",
 		},
-	})
+	}); err != nil {
+		return nil, err
+	}
+
+	authConfigsJSON, err := ioutil.ReadFile("/cnab/app/registry-creds.json")
```

Comment:
Can you extract this file as a const like `fileDockerContext` at the beginning of this file?

Reply:
I just pushed a big update which moves the names for these (and creds, and parameters) to `internal/names.go` which accomplished the same thing.